lib/processor.js

```diff
@@ -58,7 +58,7 @@ function getComment(html) {
 
     html = html.slice(commentStart.length, -commentEnd.length);
 
-    if (html.trim().slice(0, prefix.length) !== prefix) {
+    if (!html.trim().match(regex)) {
```

Comment:
Maybe `!regex.test(html.trim)` would be more efficient in a boolean context (returns boolean value, not object)?